lib/VoiceReadout.js

```diff
@@ -0,0 +1,17 @@
+require('../webOS/webOS');
+
+var
+	options = require('enyo/options');
+
+module.exports = {
+	/**
+	* Read alert message.
+	*
+	* @public
+	*/
+	readAlert: function(s) {
+		if (options.accessibility && window.webOS && window.webOS.voicereadout) {
```

Comment:
Couple questions here ... Should `readAlert()` really be a no-op if `options.accessibility` is false? Seems like a programmatic invocation like this puts the onus on the app dev to know what they're doing and would always do something. Second, are the webOS checks necessary if we've `require()`'d the webOS lib above? Are platforms that would not have that feature?